posts/interaction-media-features.md

```diff
@@ -0,0 +1,11 @@
+feature: Interaction media features (hover, pointer, any-hover, any-pointer)
```

Comment:
Is the last part of the title (the piece in parenthesis) needed? This way the title appears to be a little long.